CMakeLists.txt

```diff
@@ -117,7 +159,6 @@ endif()
 
 set(USE_ZLIB ON)
 add_definitions( -DUSE_ZLIB )
```

Comment:
You have removed add_definitions( -DHAVE_AES ) ? Is this ok ?

Reply:
added back